lib/nano.js

```diff
@@ -458,6 +458,10 @@ module.exports = exports = nano = function dbScope(cfg) {
 
       return relax({db: dbName, doc: docName, qs: qs}, callback);
     }
+	
+	function findDoc(query, callback) {
+		return relax({ db: dbName, path: '_find', method: 'POST', body: query }, callback);
```

Comment:
none of the other methods here do this at this point, I assume `relax()` does this already.